tests/plugins/pastefromword/generated/_fixtures/Shape_adjacent_image/expected.html

```diff
@@ -0,0 +1 @@
+<p style="margin-left:0in; margin-right:0in"></p><table style="width:100%"><tbody><tr><td><div><p>hello world</p></div></td></tr></tbody></table><p><br /></p><p><br /></p><table align="left"><tbody><tr><td><br /></td><td><br /></td><td><br /></td></tr><tr><td><br /></td><td colspan="2" valign="top"><img data-cke-is-shape="true" src="file:///c:\users\foobar\appdata\local\temp\msohtmlclip1\01\clip_image001.png" style="height:106px; width:348px" /></td></tr><tr><td><br /></td></tr><tr><td><br /></td><td valign="top"><img data-cke-is-shape="true" src="file:///c:\users\foobar\appdata\local\temp\msohtmlclip1\01\clip_image002.png" style="height:106px; width:120px" /></td></tr></tbody></table><p style="margin-left:0in; margin-right:0in"><br /></p><p style="margin-left:0in; margin-right:0in"><br /></p><p style="margin-left:0in; margin-right:0in"><br /></p><p style="margin-left:0in; margin-right:0in"><br /></p><p style="margin-left:0in; margin-right:0in"><br /></p><p style="margin-left:0in; margin-right:0in"><br /></p><p style="margin-left:0in; margin-right:0in"><br /></p><p><br /></p>
```

Comment:
This test work a little bit different.
1. Textbox is treat as a shape. So there is 4 shapes in source doc.
2. Because 3 first shapes are adjacent, they are treat as a single img in the output. Take a look here: https://github.com/ckeditor/ckeditor-dev/blob/534af88e2b35d7e4906c38b44957940d77e2c5d8/tests/plugins/pastefromword/generated/_fixtures/Shape_adjacent_image/word2013/chrome.html#L1084
It shows that this single `img` tag belong to 3 shapes with given names.
3. If you remove textbox and remain other shapes untouched, this will generate 3 img tags in the output, because 3 top shapes will stop being adjacent.

Reply:
Makes sense, haven't noticed that detail. So this means the expected output is valid in this case, isn't it? 👍

Reply:
I think it's valid here :)